app.legcord.Legcord.yml

```diff
@@ -0,0 +1,61 @@
+app-id: app.legcord.Legcord
+
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+
+command: startlegcord
+separate-locales: false
+
+finish-args: # We aren't trying to get tray icons working here for several reasons, mainly I'm too stressed to figure them out. -Oro
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications # Discord push notifs
+  - --talk-name=org.kde.StatusNotifierWatcher # Tray functionalities on KDE
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: minerals
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 minerals/startlegcord /app/bin/startlegcord
+      - install -D minerals/app.legcord.Legcord.desktop -t /app/share/applications/
```

Comment:
you mean the binary package doesn't have a desktop file?

Reply:
It does, but that desktop file is only meant for appimage and has some special flags for it. I want to use own one here for `startlegcord` wrapper. Upstream allows it